### ISSUE TITLE
Allow editing projects by double clicking on Title

### ISSUE THREAD

**Issue by rohmishra** [iman-salmani/iplan#4]
Currently, any changes to a project can only be performed by the way of edit project option in the menu. A more intuitive way for this would be to allow editing name by clicking or tapping on it on the top bar similar to how lists can be renamed.

**Comment by iman-salmani** [comment]
good idea. i will add this in next commits along side open project edit window when new project button clicked. but for now let me fix memory leak before adding more code.

**Comment by iman-salmani** [comment]
Feature added.

**Comment by iman-salmani** [comment]
Feature added.
